Library/Formula/docker-machine.rb

```diff
@@ -3,8 +3,8 @@
 class DockerMachine < Formula
   desc "Create Docker hosts locally and on cloud providers"
   homepage "https://docs.docker.com/machine"
-  url "https://github.com/docker/machine/archive/v0.5.6.tar.gz"
-  sha256 "b6dee30f6e8eb1d3f130cbd9545018e0abb514a968bcd1acd6006ae343f8f402"
+  version "0.5.6"
+  url "https://github.com/docker/machine.git", :revision => "v0.5.6"
```

Comment:
Yes. The Makefile uses the git sha1 so that `docker-machine version` shows the exact version built.
I'll update the Formula